scripts/run.ps1

```diff
@@ -68,11 +68,19 @@ function Install() {
             Invoke-Expression ($certbotExp -f $qFlag)
         }
     }
+
+    Write-Host "(!) " -f cyan -nonewline
+    [string]$database = $( Read-Host "Enter the database name for your Bitwarden instance (ex. vault): "
+    echo ""
+
+    if ($database -eq "") {
+        $database = "vault"
+    }
     
     Pull-Setup
     docker run -it --rm --name setup -v ${outputDir}:/bitwarden bitwarden/setup:$coreVersion `
         dotnet Setup.dll -install 1 -domain ${domain} -letsencrypt ${letsEncrypt} `
-        -os win -corev $coreVersion -webv $webVersion -q $setupQuiet
+        -os win -corev $coreVersion -webv $webVersion -q $setupQuiet -dbname $database
```

Comment:
Database names can have spaces in them in SQL Server, please ensure this command parameter is text qualified when being written out or passed to the docker run command.

Reply:
Added quotes around the variable in run.sh and run.ps1. Both worked properly when tested.